### PR TITLE
Update azure and microsoft

### PR DESCRIPTION
**Pull request by KukiSa**
## 上海蓝云网络科技有限公司 沪ICP备13015306号 B1.B2-20150124
**_世纪互联蓝云_ (21 Vianet Blue Cloud)**

Microsoft Azure 由 _世纪互联_ 运营 (Microsoft Azure operated by _21Vianet_)
Microsoft Office 365 由 _世纪互联_ 运营 (Microsoft Office 365 operated by _21Vianet_)
Microsoft Dynamics 365 由 _世纪互联_ 运营 (Microsoft Dynamics 365 operated by _21Vianet_)
Microsoft Power Platform 由 _世纪互联_ 运营 (Microsoft Power Platform operated by _21Vianet_)

`chinacloudapi.cn` 沪ICP备13015306号-2
`chinacloud-mobile.cn` 沪ICP备13015306号-3
`chinacloudsites.cn` 沪ICP备13015306号-4
`windowsazurestatus.cn` 沪ICP备13015306号-5
`windowsazure.cn` 沪ICP备13015306号-6
`lync.cn` 沪ICP备13015306号-9
`microsoftonline.cn` 沪ICP备13015306号-10
`microsoftonline-i.cn` 沪ICP备13015306号-12
`microsoftonline-m.cn` 沪ICP备13015306号-14
`microsoftonline-m-i.cn` 沪ICP备13015306号-15
`microsoftonline-p.cn` 沪ICP备13015306号-16
`microsoftonline-p.net.cn` 沪ICP备13015306号-17
`microsoftonline-p-i.cn` 沪ICP备13015306号-18
`microsoftonline-p-i.net.cn` 沪ICP备13015306号-19
`office365.cn` 沪ICP备13015306号-20
`officewebapps.cn` 沪ICP备13015306号-21
`outlook.cn` 沪ICP备13015306号-22
`onmschina.cn` 沪ICP备13015306号-23
`sharepoint.cn` 沪ICP备13015306号-24
`azure.cn` 沪ICP备13015306号-25
`21vbluecloud.com` 沪ICP备13015306号-27
`21vbluecloud.net` 沪ICP备13015306号-28
`trafficmanager.cn` 沪ICP备13015306号-32
`microsoftazurestatus.cn` 沪ICP备13015306号-33
`trustcenter.cn` 沪ICP备13015306号-36
`powerbi.cn` 沪ICP备13015306号-38
`pbiwebcontent.cn` 沪ICP备13015306号-39
`azureiotsuite.cn` 沪ICP备13015306号-41
`mcchcdn.com` 沪ICP备13015306号-42
`azchcdna.com` 沪ICP备13015306号-43
`azchcdnb.com` 沪ICP备13015306号-44
`azchcdne.com` 沪ICP备13015306号-45
`azchcdnc.com` 沪ICP备13015306号-46
`azchcdnd.com` 沪ICP备13015306号-47
`azchcdnf.com` 沪ICP备13015306号-48
`azchcdng.com` 沪ICP备13015306号-49
`azchcdni.com` 沪ICP备13015306号-50
`azchcdnh.com` 沪ICP备13015306号-51
`azure-api.cn` 沪ICP备13015306号-52
`azure-apihub.cn` 沪ICP备13015306号-53
`azk8s.cn` 沪ICP备13015306号-54
`azurerms.cn` 沪ICP备13015306号-55
`aadrm.cn` 沪ICP备13015306号-56
`microsofttranslator-int.cn` 沪ICP备13015306号-57
`azurehdinsight.cn` 沪ICP备13015306号-58
`dynamics.cn` 沪ICP备13015306号-59
`aggresmart.com` 沪ICP备13015306号-60
`azchcdnl.com` 沪ICP备13015306号-61
`azchcdnk.com` 沪ICP备13015306号-62
`azchcdnj.com` 沪ICP备13015306号-63
`azchcdnm.com` 沪ICP备13015306号-64
`azchcdnn.com` 沪ICP备13015306号-65
`azchcdno.com` 沪ICP备13015306号-66
`azchcdnp.com` 沪ICP备13015306号-67
`azchcdnq.com` 沪ICP备13015306号-68
`azchcdnr.com` 沪ICP备13015306号-69
`azchcdns.com` 沪ICP备13015306号-70
`msftauth.cn` 沪ICP备13015306号-71
`msauth.cn` 沪ICP备13015306号-72
`msauthimages.cn` 沪ICP备13015306号-73
`msftauthimages.cn` 沪ICP备13015306号-74
`msidentity.cn` 沪ICP备13015306号-75
`bluecloudprod.com` 沪ICP备13015306号-76
`azurecr.cn` 沪ICP备13015306号-77
`azcrmc.cn` 沪ICP备13015306号-78
`azcrmc-test.cn` 沪ICP备13015306号-79
`azurecr-test.cn` 沪ICP备13015306号-80
`aztask.cn` 沪ICP备13015306号-81
`21vbc.com` 沪ICP备13015306号-82
`blueaggrestore.com` 沪ICP备13015306号-83
`appserviceenvironment.cn` 沪ICP备13015306号-84
`azure-devices-provisioning.cn` 沪ICP备13015306号-85
`management-azure-devices-provisioning.cn` 沪ICP备13015306号-86
`azure-automation.cn` 沪ICP备13015306号-87
`b2clogin.cn` 沪ICP备13015306号-88
`mgmt-azure-api.cn` 沪ICP备13015306号-89
`azure-dns.cn` 沪ICP备13015306号-90
`powerapps.cn` 沪ICP备13015306号-91
`hdinsightservices.cn` 沪ICP备13015306号-92
`msftcloudes.cn` 沪ICP备13015306号-93
`management-azure-devices.cn` 沪ICP备13015306号-94
`azure-devices.cn` 沪ICP备13015306号-95
`mncmsidlab1.cn` 沪ICP备13015306号-96
`powerautomate.cn` 沪ICP备13015306号-97
`msidlabpbmc.cn` 沪ICP备13015306号-98
`azuremresolver.cn` 沪ICP备13015306号-99
`azureprivatedns.cn` 沪ICP备13015306号-100
`unity3dcloud.cn` 沪ICP备13015306号-101
`azure-dns-1.cn` 沪ICP备13015306号-102
`office365-net.cn` 沪ICP备13015306号-103
`azure-dns-4.cn` 沪ICP备13015306号-104
`azure-dns-2.cn` 沪ICP备13015306号-105
`azure-dns-3.cn` 沪ICP备13015306号-106
`apihub-internal.cn` 沪ICP备13015306号-107
`msappproxy.cn` 沪ICP备13015306号-108
`microsoftmetrics.cn` 沪ICP备13015306号-109
`aznbcontent.cn` 沪ICP备13015306号-110
`azuresandbox.cn` 沪ICP备13015306号-111
`cegid-cloud.cn` 沪ICP备13015306号-112
`b3itech.cn` 沪ICP备13015306号-114
`azure-connectedvehicles.cn` 沪ICP备13015306号-115
`azure-connectedvehicles-stage.cn` 沪ICP备13015306号-116
`mschcdn.com` 沪ICP备13015306号-117
`powerappsportals.cn` 沪ICP备13015306号-118

## Supplementary Instructions
1. About adding `management-azure-devices.net` and `management-azure-devices-int.net`, please consult the certs:
- https://crt.sh/?id=7641451710
- https://crt.sh/?id=7637475025

2. `trafficmanager.net` is used by the Azure Traffic Manager service, so it should be in the file `azure`.

### CODE REVIEW

File: data/azure [hunk 1]
```diff
@@ -1,10 +1,27 @@
 # All .azure domains
 azure
 
-azure-dns.com
-azure-dns.info
-azure-dns.net
-azure-dns.org
```
Comment (rootmelo92118) [inline data/azure:7]: why did you remove those domains?

Comment (KukiSa) [reply data/azure:7]: > why did you remove those domains?

They are moved to new line 109.

**Review by rootmelo92118 — APPROVED** [review]
